### DISCUSSION TITLE
Account based preferences

### DISCUSSION THREAD

**Discussion by ShreyanKhanna** [For-0/vocabustudy#190] [Ideas]
I think it would be a good idea to make it so that the user can change how the Vocabustudy page looks for them such as colors, backgrounds, and patterns(of course we can keep things like the homepage the same)  as well as control how many drag & drops, questions, etc. they want. I think that we can have this whole thing in the setting as well as move the light/dark mode there as well. I know it may be too early but just putting it out there for the future.

**Comment by grimsteel** [comment]
Make an issue with the enhancement label

**Reply by ShreyanKhanna** [reply]
ok

**Comment by IMGROOT2** [comment]
I feel like this could be implemented in time for 1.0.0, but probably 2.0.0. I feel we should keep the appearance preferences at the bottom right so customizing is easier (monkeytype is a great example of the versatility and simplicity of a well-thought-out interface). 

But yeah, everything sounds good
